packages/ngx-deploy-npm/src/generators/install/schema.json

```diff
@@ -4,20 +4,24 @@
   "title": "",
   "type": "object",
   "properties": {
-    "projects": {
-      "description": "Which libraries should configure",
-      "type": "array",
-      "items": {
-        "type": "string"
-      }
+    "project": {
+      "description": "Which library should configure. This will add the target deploy to that project. It must be a publishable library.",
+      "type": "string",
+      "x-prompt": "Which library should configure?"
+    },
+    "distFolderPath": {
+      "type": "string",
+      "description": "The dist folder path. The path should be relative to the project's root",
+      "$default": {
+        "$source": "argv",
```

Comment:
Review this default to be working as expected

Reply:
It shows the workspace projects 
![image](https://github.com/bikecoders/ngx-deploy-npm/assets/7026066/624f1728-c58b-4158-bea8-d8f7d7f517cc)

Reply:
Removing it since doesn't seems user friendly to use the first parameter as the dist folder